source/_posts/azure-purview.md

```diff
@@ -0,0 +1,149 @@
+
+Azure Purview is an update to the existing Azure Data Catalog Gen 2 service. Azure Data catalog Gen 2 focused more on managing metadata instead of real data governance and was capable of mapping, searching for, and tagging data within data sources but wasn't able to classify its data.This made it hard to be compliant to different laws pertaining data protection, like GDPR. 
+
+Azure Purview allows you to answer the following questions:
+
+- What kind of data do I have?
+- What are all the different tables and fields in the database? 
+- Where is any sensitive data? 
+
+# How does Azure Purview work?
+
+It all starts with the data. Organisations can have many different data assets like files, tables,reports and databases. These data sources can exist across different environments, like in the public cloud, on-premise, and SaaS environments. Azure Purview even supports multi-cloud. Because of this support, you do not have to move data from on-premise or other cloud providers to Azure: everything you want to scan can stay where it is. 
+
+<img src="/images/azure-purview/cloud-model.png" />
+
+You can then connect these different environments with Azure Purview via the different connectors that are available. 
+
+Purview can scan these data sources to extract different things like metadata, lineage and data classification. The scans operate serverless, so you only pay for what you use. 
+
+When the scanning is done, all the data that is found is published to the Azure Purview datamap. The data map is like a graph describing all the data and relationships between data assets across different environments.The more data you scan, the more powerful and useful this graph becomes!
+
+## Business Context, Classification & Lineage
+
+Once the scan is complete, the data users in your company can start using the data that was just scanned. One thing they can do is add business context to the different fields
+
+<img src="/images/azure-purview/business-context.png" />
+
+In the example you can see that you can expand on the data within your organistation by adding descriptions and context to them. This allows an organisation to define any domain knowledge or business context to their data users in a clear way. 
+
+Secondly, Classification is another important part of Azure Purview. It's important for an organisation to be compliant with rules and regulations like GDPR. Azure Purview helps with this by classifying data fields. It uses AI to see what a field contains, and classifies it accordingly. With this knowledge, you know where within your company your sensitive data resides. 
+
+Last but not least, Azure Purview offers Data Lineage. Data Lineage lets you see where your data comes from and where you're data is going to. The image below shows the flow of data from the salesOrderHeader table to different areas within the organisation. Some operations are applied to the data, which is then transformed and shown in PowerBI. 
```

Comment:
consider writing it as data lineage instead (per https://docs.microsoft.com/en-us/azure/purview/concept-data-lineage Microsoft does not capitalize it)
comes from >> originates
you're >> your 
going to >> headed
consider adding markup to salesOrderHeader (code block?)